python/paddle/v2/framework/tests/op_test.py

```diff
@@ -168,7 +168,11 @@ def get_gradient(scope, op, inputs, outputs, grad_name, place,
 class OpTest(unittest.TestCase):
     def check_output_with_place(self, place):
         self.scope = core.Scope()
-        self.op = create_op(self.scope, self.op_type, self.inputs, self.outputs)
+        op_inputs = self.inputs if hasattr(self, "inputs") else dict()
+        op_outputs = self.outputs if hasattr(self, "outputs") else dict()
```

Comment:
When the users could not set `self.inputs`  and `self.outputs` ?  Aren't they always needed?

Reply:
Some operator, like random operator has no input. NOP has no input and output. 
I will remove self.outputs here.